docs/tutorials/IndexerMigration.md

```diff
@@ -126,3 +125,155 @@ graph LR;
+Sets the sync round to the provided parameter. The provided sync round cannot be lower than the current sync round.
+If no sync round is currently set, the round must be within `MaxAcctLookback` of the ledger's current round at the time
+of the API call. Providing an invalid round will return a `400 Bad Request`.
+
+### State Deltas
+The algod `block/{round}` endpoint has always been available to retrieve the block header and the transactions which
+were validated during a given round, but Indexer also needs to know the side effects of transaction execution--account
+balance updates, app global and local state changes, etc.
+
+Indexer currently uses a local ledger to get this information, a stripped down version of an algod node which stores
+balance information locally, and evaluates transactions in order to determine changes made without having to do an
+extra database roundtrip.
+
+We've created a new API on algod, `GET v2/deltas/{round}`, which exposes per-round deltas. It is a direct serialization
+of the data which is stored in the cache in algod, and it is backward compatible with the postgresql serialization that
+the Indexer database currently uses.
+
+Now instead of running a local ledger, all that is required to run an Indexer pipeline via Conduit is a Follower algod
+node. Conduit's algod importer plugin will coordinate usage of both the sync round APIs and the deltas API in order to
+fetch all required data directly from the algod node, and ensure that the node is kept in sync with the pipeline round.
+
+
+## Altering Architecture of Indexer Applications
+
+Let's take a look at a theoretical application more closely and see what in particular we need to do to alter it to use
+Conduit. 
+
+Here's a diagram of a containerized application (leaving out some details such as load balancers, and scaled out API
+nodes).
+
+```mermaid
+graph LR;
+    algod["Archival Alogd"]
+    indexer["Indexer"]
+    psql["Postgresql"]
+    subgraph "Persistent Volume"
+        ll["Local Ledger"];
+    end
+    
+    indexer-->algod;
+    indexer-->psql;
+    indexer-->ll;
+```
+
+### Step 1: Convert from Archival to Follower Node
+In our hypothetical existing architecture, we likely have a persistent volume storing the algod data directory for our
+Archival node. It is probably still a good idea to maintain the algod's data in a persistent volume, though the amount
+of storage required will be minimal in comparison.
+
+
+If you have an existing Archival node which is on the same round as your Indexer, you can simply stop algod, alter your
+config as follows, and restart it.
+```yaml
+Archival: false,
+MaxAcctLookback: 256,
+CatchupParallelBlocks: 32,
+EnableFollowMode: true
+```
+Your node will then drop all but the last 1000 rounds from your database, and will have `2 * MaxAcctLookback`
+rounds in its cache.
+
+However, if your algod node's round is too far beyond your Indexer's you will need to catchup your node from scratch.
```

Comment:
how is a user supposed to detect this?

Reply:
There's an error message in the log
> ledger state delta not found: node round (%d), required round (%d): verify follower node configuration and ensure follower node has its sync round set to the required round, re-deploying the follower node may be necessary

Reply:
Also updated with info below on how to use `goal` for algod and `/health` for indexer to determine which round they are on.